Guides/Migrating.md

```diff
@@ -40,7 +40,15 @@ Also update dependencies version to latest if required.
 
 And run pod install. If successful, running and testing will work correctly. 
 
-#### 9. In example project, add the newly added destination by setting its relative path.
+#### 9. For tests to compile successfully from command line, code signing for the test targets must be set. (It's enough that it's a personal team, Apple ID does not need to have purchased developer program)
+
+#### 10. It may happen that the pod install has created extra schemes in workspace, simply delete them, but (choose Remove reference) upon delete confirmation.
+
+#### 11. It may also happen that the test targets have not been created, in that case, select scheme -> edit scheme ->  test -> +  and it will automatically create a test scheme.
```

Comment:
Again once they navigate to test what specifically are they hitting? For me it seems it the test scheme is already enabled so I can't see the workflow for when its not automatically enabled. But can we be as specific as to say be on the `Info` tab and hit `+`. Then choose the test target to add to this scheme based off the integration which you just migrated. Again the more explicit here the better.